README.md

```diff
@@ -11,7 +11,7 @@ Learn how to use **`Alpine.js`** to build **declarative + responsive UI _fast_**
 We heard a lot about **`Alpine.js`** 
 [online](https://github.com/dwyl/technology-stack/issues/87) 
 and wanted to know what the fuss was about. <br />
-We weren't dissapointed.
+We weren't disapointed.
```

Comment:
We were _both_ misspelling it ... 😜 
https://www.google.com/search?q=disappointed

```suggestion
We weren't disappointed.
```

![image](https://user-images.githubusercontent.com/194400/200318948-f8f962da-45ac-4e70-abbb-1fc2718bc31a.png)

Reply:
Thanks for double checking this one!